mathesar_ui/src/stores/tableData.ts

```diff
@@ -519,6 +526,29 @@ export function getTable(db: string, id: number, options?: Partial<TableOptionsD
   return table;
 }
 
+export function updateColumnPosition(cols:TableColumn[], position:ColumnPosition):ColumnPosition {
+  return calculateColumnPosition(cols, position);
+}
```

Comment:
This method does not perform anything additionally. We could remove this and rename `calculateColumnPosition` to `updateColumnPosition` instead.